docs/tempo/website/configuration/_index.md

```diff
@@ -456,11 +458,28 @@ compactor:
 ```
 
 ## Storage
+
+Tempo supports Amazon S3, GCS, Azure, local file system, and Parquet for storage. In addition, you can use Memcached or Redis for increased query performance.
+
 For more information on configuration options, see [here](https://github.com/grafana/tempo/blob/main/tempodb/config.go).
 
-The storage block is used to configure TempoDB. It supports S3, GCS, Azure, local file system, and optionally can use Memcached or Redis for increased query performance.
+### Local storage recommendations
+
+While you can use local storage, object storage is recommended for production workloads.
+Tempo is designed for object storage more than local storage.
+
+At Grafana Labs, we have almost exclusively run Tempo with SSDs instead of hard drives when using local storage.
+Hard drives should work if the read and write speeds are good. There may be more latency in I/O operations.
```

Comment:
> Hard drives should work if the read and write speeds are good. There may be more latency in I/O operations.

I'm not a fan of this statement. Specially 'good' — what are _good_ read/write speeds? I think we lack experience running local storage for real workloads to comment on it on official documentation. I'd try to make it more vague or remove this part. But I'd like to hear more thoughts on this.

Reply:
If we lack experience running local storage for real workloads, then we should probably remove this statement. I could change it to say this: 
"At Grafana Labs, Tempo has been tested with SSDs as local storage. Hard drives have not been tested."